example/App.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+import React, {useState, useEffect} from 'react';
+import {
+  ScrollView,
+  Button,
+  TextInput,
+  StyleSheet,
+  PermissionsAndroid,
+  Platform,
+  View,
+  Permission,
+} from 'react-native';
+
+import {useJellyfishClient} from '@jellyfish-dev/react-native-client-sdk';
+
+import {Room} from './src/Room';
+import {JELLYFISH_URL} from '@env';
+
+function App(): JSX.Element {
+  const {connect, join, cleanUp} = useJellyfishClient();
```

Comment:
please handle the errors, just displaying them as text somewhere is ok, I was confused when testing when I couldn't connect and the app didn't do anything, just printed in the logs "socket closed"

Reply:
Added error message on the top of the screen